core/src/main/java/com/graphhopper/storage/BaseGraph.java

```diff
@@ -175,51 +172,45 @@ void checkInitialized() {
             throw new IllegalStateException("The graph has not yet been initialized.");
     }
 
-    protected int loadNodesHeader() {
-        nodeEntryBytes = nodes.getHeader(1 * 4);
-        nodeCount = nodes.getHeader(2 * 4);
-        bounds.minLon = Helper.intToDegree(nodes.getHeader(3 * 4));
-        bounds.maxLon = Helper.intToDegree(nodes.getHeader(4 * 4));
-        bounds.minLat = Helper.intToDegree(nodes.getHeader(5 * 4));
-        bounds.maxLat = Helper.intToDegree(nodes.getHeader(6 * 4));
+    private void loadNodesHeader() {
+        nodeEntryBytes = nodes.getHeader(0 * 4);
+        nodeCount = nodes.getHeader(1 * 4);
+        bounds.minLon = Helper.intToDegree(nodes.getHeader(2 * 4));
+        bounds.maxLon = Helper.intToDegree(nodes.getHeader(3 * 4));
+        bounds.minLat = Helper.intToDegree(nodes.getHeader(4 * 4));
+        bounds.maxLat = Helper.intToDegree(nodes.getHeader(5 * 4));
 
         if (bounds.hasElevation()) {
-            bounds.minEle = Helper.intToEle(nodes.getHeader(7 * 4));
-            bounds.maxEle = Helper.intToEle(nodes.getHeader(8 * 4));
+            bounds.minEle = Helper.intToEle(nodes.getHeader(6 * 4));
+            bounds.maxEle = Helper.intToEle(nodes.getHeader(7 * 4));
         }
 
-        frozen = nodes.getHeader(9 * 4) == 1;
-        return 10;
```

Comment:
The return value of these methods was never used anyway, they can also be private.